src/global-firestore-data/override-fulfillment-choices.ts

```diff
@@ -39,30 +60,74 @@ export const toggleRequirementChoice = (
     }
   });
 
-export const updateRequirementChoices = (
-  updater: (
-    oldChoices: FirestoreOverriddenFulfillmentChoices
-  ) => FirestoreOverriddenFulfillmentChoices
+export const deleteCoursesFromRequirementChoices = (
+  courseUniqueIds: readonly (string | number)[]
 ): void => {
-  overriddenFulfillmentChoicesCollection
-    .doc(store.state.currentFirebaseUser.email)
-    .set(updater(store.state.overriddenFulfillmentChoices));
+  const courseUniqueIdStrings = new Set(courseUniqueIds.map(uniqueId => uniqueId.toString()));
+  updateRequirementChoices(choices =>
+    Object.fromEntries(
+      Object.entries(choices).filter(([uniqueId]) => !courseUniqueIdStrings.has(uniqueId))
+    )
+  );
 };
```

Comment:
The diff on Github doesn't match up exactly, but same idea here. Refactored to use `updateRequirementChoices`.